### PR TITLE
Cleanup of CMakeLists.txt

### PR DESCRIPTION
**Pull request by zalox**
@eidheim Please review and merge if ok!

### CODE REVIEW

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -41,129 +44,101 @@ else()
...
-  ${LIBLLDB_LIBRARIES}
-  #${PYTHON_LIBRARIES}
-)
+include_directories(${global_includes})
+add_executable(${project_name} ${project_files})
+target_link_libraries(${project_name} ${global_libraries})
 
-#  install(TARGETS ${project_name} ${module}
-install(TARGETS ${project_name}
+install(TARGETS ${application_name}
```
Comment (eidheim) [inline src/CMakeLists.txt]: Make install does not install the juci binary. 

I tried `message("${application_name}")` outputs "" on OS X (only tested here), while `message("${project_name}")` outputs "juci" correctly. I guess you intended to write `install(TARGETS ${project_name}` here or?

Comment (zalox) [inline src/CMakeLists.txt]: Yes, I'll fix it!

Comment (zalox) [inline src/CMakeLists.txt]: Fixed!

File: src/CMakeLists.txt [hunk 2]
```diff
@@ -41,129 +44,101 @@ else()
...
-  ${GTKMM_LIBRARIES}
-  ${GTKSVMM_LIBRARIES}
-  ${Boost_LIBRARIES}
-  ${ASPELL_LIBRARIES}
-  ${LIBLLDB_LIBRARIES}
-  #${PYTHON_LIBRARIES}
-)
+include_directories(${global_includes})
+add_executable(${project_name} ${project_files})
+target_link_libraries(${project_name} ${global_libraries})
```
Comment (eidheim) [inline src/CMakeLists.txt]: Nice you cleaned up these two files! 

Although, the variables `global_includes` and `global_libraries` are only used once. Maybe better to just use` include_directories` and `target_link_libraries` on the various `${..._INCLUDE_DIRS}` and `${..._LIBRARIES}` directly then (like before)?

Comment (zalox) [inline src/CMakeLists.txt]: list(APPEND global_includes)

I was thinking we might need that in the future.

Comment (eidheim) [inline src/CMakeLists.txt]: Good point, I'll merge it Tomorrow morning or so after testing on all the OS'es.

**Pull request merged**